eng/testing/MobileRunnerTemplate.sh

```diff
@@ -0,0 +1,37 @@
+﻿#!/usr/bin/env bash
+
+EXECUTION_DIR=`dirname $0`
```

Comment:
nit: use $() instead:

```suggestion
EXECUTION_DIR=$(dirname $0)
```